deis-logger/manifests/deis-logger-rc.yaml

```diff
@@ -25,3 +23,15 @@ spec:
         - containerPort: 1514
           name: transport
           protocol: UDP
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 8088
+          initialDelaySeconds: 1
+          timeoutSeconds: 1
+        readinessProbe:
+          httpGet:
+            path: /healthz
+            port: 8088
+          initialDelaySeconds: 1
+          timeoutSeconds: 1
```

Comment:
Well the comment location shifted around, wtf... but look at the diff. Missing EOL at the end of the file is all